static/app/components/events/eventEntry.tsx

```diff
@@ -114,13 +119,23 @@ function EventEntry({
       );
 
     case EntryType.BREADCRUMBS: {
+      const replay = !isShare && replayId && (
+        <EventReplay
+          orgSlug={organization.slug}
+          projectSlug={projectSlug}
+          replayId={replayId}
+          event={event}
+        />
+      );
+
       return (
         <Breadcrumbs
           data={entry.data}
           organization={organization as Organization}
           event={event}
           router={router}
           route={route}
+          replay={replay}
```

Comment:
this seems like the wrong prop to be passing in... the replay depends on a few props that are basically all available inside <Breadcrumbs> already (event, org) the only prop needed is `isShared`.
The the logic isn't spread out as much.